src/status_im2/common/home/title_column/view.cljs

```diff
@@ -11,7 +11,8 @@
    [rn/view {:flex 1}
     [quo/text style/title-column-text
      label]]
-   [plus-button/plus-button
-    {:on-press            handler
-     :accessibility-label accessibility-label
-     :customization-color customization-color}]])
+   (when handler
+     [plus-button/plus-button
+      {:on-press            handler
+       :accessibility-label accessibility-label
+       :customization-color customization-color}])])
```

Comment:
I made the plus-button optional so I could reuse this component in the browser screen. If you believe it's not convenient I can revert and create a new component only for browser

Reply:
hmm, I think this is fine to add. however it looks like the designs do include in the plus button in some screens. I think it might be a gap in the designs so I asked the designers to confirm this 👍